PROTOCOL.md

```diff
@@ -1933,6 +1939,282 @@ delta-table-version: 123
 }
 ```
 
+### Read Change Data Feed from a Table
+This is the API for clients to read change data feed from a table.
+
+You can provide either version or timestamp for the start and end of change data feed. The start and end versions and timestamps are inclusive in the queries. To read the changes from a particular start version to the latest version of the table, specify only the starting version or timestamp. Either startingVersion or startingTimestamp must be provided. For either starting or ending parameter, only one parameter (version or timestamp) is supported.
```

Comment:
"Either startingVersion or startingTimestamp must be provided." this reads a bit strange. How about "A starting parameter, either startingVersion or startingTimestamp, must be provided."?

Reply:
How about:
The API supports a start parameter and and an end parameter. The start/end parameter can either be a version or a timestamp. The start parameter must be provided. If the end parameter is not provided, the API will use the latest table version for it. The parameter range is inclusive in the query.